cl/users/templates/includes/webhooks_htmx/webhooks-list.html

```diff
@@ -16,24 +16,26 @@
       </p>
     </td>
     <td>{{ webhook.url }}</td>
-    <td class="right" hx-headers='{"X-CSRFToken": "{{ csrf_token }}"}'>
+    <td class="right" hx-headers='{"X-CSRFToken": "{{ csrf_token }}"}' data-loading-states>
       <button hx-delete="{% url "webhooks-detail" pk=webhook.id format='json' %}"
               hx-confirm="Are you sure you want to delete this Webhook?"
-              class="btn btn-danger btn-xs">
-        <i class="fa fa-trash-o"></i>&nbsp;Delete</button>
+              class="btn btn-danger btn-xs" data-loading-disable>
+        <i class="fa fa-trash-o"></i>&nbsp;Delete
+        <i data-loading class="fa fa-spinner fa-spin"></i>
+      </button>
       <button class="btn btn-primary btn-xs"
               hx-get="{% url "webhooks-detail" pk=webhook.id format='html' %}"
-              hx-target="#webhooks-body">
+              hx-target="#webhooks-body" data-loading-disable>
         <i class="fa fa-pencil"></i>&nbsp;Edit</button>
     </td>
   </tr>
 {% endfor %}
 {% else %}
-  <td class="text-center empty-webhooks" colspan="5">
+  <td class="text-center empty-webhooks" colspan="5" data-loading-states>
     <p>Create a new webhook:</p>
     <button id="add-webhook" class="btn btn-primary btn-xs"
             hx-get="{% url 'webhooks-render-form' format="html" %}"
             hx-target="#webhooks-body">
       <i class="fa fa-plus"></i>&nbsp;Add webhook</button>
```

Comment:
I think we might also add the spinner and disable this button during loading the state since that request renders the form it also might be a bit slow.  
This same button it's also in [webhooks.html](https://github.com/freelawproject/courtlistener/pull/2331/files#diff-e50c4d91487fe98bf3645b7158e3f7c74b7d10096220b4475909587ff24f0ce7R93)

Reply:
Sure! We can use the extension to trigger the 'loading state' of the buttons at the same time. 

The `data-loading-target` attribute allows setting a different target to apply the loading states so, we can use this attribute to select and disable the button in [webhooks.html](https://github.com/freelawproject/courtlistener/pull/2331/files#diff-e50c4d91487fe98bf3645b7158e3f7c74b7d10096220b4475909587ff24f0ce7R93) while the request that renders the form is in flight. 

The UI would look like this: 

**Loading states with both buttons**
![add-webhook-present](https://user-images.githubusercontent.com/55959657/196488311-d07a71a3-45ae-4a99-b1a4-3d5c2b1591e9.gif)

**Loading state with one button**
![add-webhook-not-present](https://user-images.githubusercontent.com/55959657/196488774-c5125d64-e299-47b2-9d2e-59450d968a49.gif)

Please let me know what you think.

Reply:
Yeah! This looks good, I think it would solve the comments. Thank you.